uamqp/aio/_connection_async.py

```diff
@@ -179,7 +212,18 @@ def _get_next_outgoing_channel(self):
     async def _outgoing_empty(self):
         if self.network_trace:
             _LOGGER.info("<- empty()", extra=self.network_trace_params)
-        await self._send_frame(0, None)
+        try:
+            if self._can_write():
+                await self.transport.write(EMPTY_FRAME)
+                self._last_frame_sent_time = time.time()
+        except (OSError, IOError, SSLError, socket.error) as exc:
+            self._error = AMQPConnectionError(
+                ErrorCondition.SocketError,
+                description="Can not send empty frame due to exception: " + str(exc),
+                error=exc
+            )
+        except Exception:
+            raise
```

Comment:
We don't need this last clause - as that's the implicit default

Reply:
removed